reminder/module.py

```diff
@@ -92,19 +92,9 @@ async def _remind(self, item: ReminderItem):
     async def _process_text(
         self, ctx: commands.Context, datetime_str: str, text: Optional[str]
```

Comment:
You can use `, *, text: ...`, which will allow users to add the text commentary without the quotes, as `*` acts as "hungry" operator and will eat all words and place them in one long string.

Reply:
But this is already handeld in commands...

Reply:
OK, my bad.